setup.py

```diff
@@ -2,11 +2,9 @@
 
 from setuptools import setup
 
-import i18n
-
 setup(
     name='edx-i18n-tools',
-    version=i18n.__version__,
+    version='0.3.7',
```

Comment:
If you prefer to have the version here then I'd delete the one in __init__.py. I'm not sure why the convention is to have the latter file, but it is codified that way in the cookiecutters.